src/components/forms/NetworkDevicesForm/InheritedNetworkRow.tsx

```diff
@@ -0,0 +1,106 @@
+import ResourceLink from "components/ResourceLink";
+import { getConfigurationRowBase } from "components/ConfigurationRow";
+import ReadOnlyAclsList from "components/forms/NetworkDevicesForm/ReadOnlyAclsList";
+import type { InstanceAndProfileFormikProps } from "components/forms/instanceAndProfileFormValues";
+import type { InheritedNetwork } from "util/configInheritance";
+import type { LxdNetwork } from "types/network";
+import type { MainTableRow } from "@canonical/react-components/dist/components/MainTable/MainTable";
+import type { EditInstanceFormValues } from "pages/instances/EditInstance";
+import classnames from "classnames";
+import NetworkDevice from "./NetworkDevice";
+import type { LxdNicDevice, LxdNoneDevice } from "types/device";
+
+interface Props {
+  device: InheritedNetwork;
+  project: string;
+  managedNetworks: LxdNetwork[];
+  formik: InstanceAndProfileFormikProps;
+  index: number;
+}
+
+export const getInheritedNetworkRow = ({
+  device,
+  project,
+  managedNetworks,
+  formik,
+  index,
+}: Props): MainTableRow => {
+  const readOnly = (formik.values as EditInstanceFormValues).readOnly;
+  const isOverridden = formik.values.devices
+    .map((t) => t.name)
+    .includes(device.key);
+  const overrideDevice = formik.values.devices.find(
+    (t) => t.name === device.key,
+  ) as LxdNicDevice | LxdNoneDevice | undefined;
+
+  const overrideNetwork = managedNetworks.find(
+    (t) => t.name === (overrideDevice as LxdNicDevice)?.network,
+  );
+
+  return getConfigurationRowBase({
+    configuration: (
+      <b
+        className={classnames({
+          "u-text--muted": isOverridden,
+          "u-text--line-through": isOverridden,
+        })}
+      >
+        {device.key}
+      </b>
```

Comment:
We should only mute and line through if this is overridden with the none device. When overriding with another network, this should be displayed normally.